include/EditView/Panels.js

```diff
@@ -1,6 +1,9 @@
 /*********************************************************************************
```

Comment:
Dear Matt


This is changed, i will pay attention on this from now.


Jerzy

________________________________
From: Matt Lorimer <notifications@github.com>
Sent: 07 July 2016 15:50:04
To: salesagility/SuiteCRM
Cc: Jerzy Brzezinski; Author
Subject: Re: [salesagility/SuiteCRM] Adding Events Delegates Filter via Subpanel (#1709)


In include/EditView/Panels.js<https://github.com/salesagility/SuiteCRM/pull/1709#discussion_r69920751>:

> @@ -1,6 +1,9 @@
>  /*********************************************************************************


this is the old licence header

-
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/salesagility/SuiteCRM/pull/1709/files/daa22c19c56a4c1de1cf1a62a308172c75944a2c..52106795574d6815ea24610072da1de8ec12a121#r69920751>, or mute the thread<https://github.com/notifications/unsubscribe/AKP8DF7q68VCLKMcN3wQTKVZjRCYslgbks5qTRIcgaJpZM4JF9xd>.